packages/calling-stateful-client/package.json

```diff
@@ -38,7 +38,7 @@
     "@azure/communication-calling": "1.12.0-beta.2 || >=1.4.4"
```

Comment:
If we want to update the base version, then we need to update this as well.

Reply:
I must have missed this one I will change this one as well

Reply:
should be all updated now. I did a search for "1.12.0-beta.2 || >=1.4.4" and replaced with "1.12.0-beta.2 || >=1.11.1"